js/src/components/paid-ads/audienceSection.test.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import AudienceSection from '.~/components/paid-ads/audience-section';
+
+jest.mock( '.~/hooks/useCountryKeyNameMap', () =>
+	jest.fn( () => ( {
+		GB: 'United Kingdom',
+		US: 'United States',
+		ES: 'Spain',
+	} ) )
+);
+
+jest.mock( '.~/hooks/useTargetAudienceFinalCountryCodes', () =>
+	jest.fn( () => ( { data: [ 'GB', 'US', 'ES' ] } ) )
+);
+
+describe( 'audienceSection', () => {
+	const defaultProps = {
+		formProps: {
+			getInputProps: () => ( {
+				checked: true,
+				className: null,
+				help: null,
+				onBlur: ( f ) => f,
+				onChange: undefined,
+				selected: [ 'GB' ],
+				value: [ 'GB' ],
+			} ),
+		},
+		countrySelectHelperText:
+			'Once a campaign has been created, you cannot change the target country.',
+	};
```

Comment:
💅 

Except if those default props are there for some reason I would suggest to have it as neat as possible. 

```
const defaultProps = {
		formProps: {
			getInputProps: () => ( {} ),
		},
	};
```

Reply:
Thanks, Miguel, I have done some cleaning on the defaultProps, it is now only using the required params to run the test. 

I also added some code to check if the onChange function has been called or not. Like this, we are covering more parts of the code.  

If It looks good to you, I will proceed with the merge!